lib/open_project/omni_auth/authorization.rb

```diff
@@ -109,6 +109,7 @@ def self.authorize_user_for_provider(provider, &block)
       def self.after_login(&)
         add_after_login_callback AfterLoginBlockCallback.new(&)
       end
+      OpenProject::Deprecation.deprecate_method(self, :after_login)
```

Comment:
Is there a way to add a pointer to the new OpenProject callback/hook that replaced this? I always find it super frustrating when a method is deprecated without any further hints. Especially when there is indeed a substitute.

Reply:
I've moved this into the method to define the warning message, that makes the reason clearer.